nmt_chainer/beam_search.py

```diff
@@ -100,12 +108,12 @@ def compute_next_lists(new_state_ensemble, new_scores, beam_width, eos_idx, curr
         update_next_lists(num_case, idx_in_case, new_cost, eos_idx, new_state_ensemble, 
                           finished_translations, current_translations, current_attentions,
            next_states_list, next_words_list, next_score_list, next_translations_list, 
-           attn_ensemble, next_attentions_list, need_attention = need_attention)
+           attn_ensemble, next_attentions_list, current_states_sum, next_states_sum_list, need_attention = need_attention)
```

Comment:
Actually, how/ why is this current_states_sum, next_states_sum_list thing used ? It is for multilingual thing, I suppose. But I could not understand from the code.